examples/step-21/doc/intro.dox

```diff
@@ -76,8 +76,22 @@ can solve right away with the techniques of the first few tutorial programs
 similar). However, we have not said anything yet about the saturation, which
 of course is going to change as the fluids move around.
 
-The second part of the equations is a therefore description of the
-dynamics of the saturation. We model this as an advected quantity:
+The second part of the equations is the description of the
+dynamics of the saturation. The saturation equation for the displacing fluid (water) is:
+@f{eqnarray*}
+  S_{t} + \nabla \cdot (F(S) \mathbf{u}) = \{q}_{w},
```

Comment:
That's not going to compile. I think you need to write this as follows:
```suggestion
  S_{t} + \nabla \cdot (F(S) \mathbf{u}) = q_{w},
```
There are numerous other places below where you have backslashes as in `\q` and `\{` that are not proper latex.